styles/GlobalStyle.js

```diff
@@ -87,21 +77,53 @@ const GlobalStyle = createGlobalStyle`
   }
 
   span.task {
-    background-color: black;
    margin-top: 0.5rem;
+   &:hover{
+     cursor: pointer
+   }
   }
 
   span.accordion {
     overflow: hidden;
     white-space: nowrap;
     text-overflow: ellipsis;
     margin-top: 0.5rem;
-
+    &:hover{
+      cursor: pointer
   }
 
   span.dueDate{
     margin-left: auto
   }
+/* 
+  input[type="checkbox"] {
+  appearance: none;
+  background-color: #fff;
+  margin: 0;
+  font: inherit;
+  color: currentColor;
+  width: 1.15em;
+  height: 1.15em;
+  border: 0.15em solid currentColor;
+  border-radius: 0.15em;
+  transform: translateY(-0.075em);
+  display: grid;
+  place-content: center;
+}
+
+input[type="checkbox"]::before {
+  content: "";
+  width: 0.65em;
+  height: 0.65em;
+  transform: scale(0);
+  transition: 120ms transform ease-in-out;
+  box-shadow: inset 1em 1em var(--form-control-color);
+}
+
+input[type="checkbox"]:checked::before {
+  transform: scale(1);
+} */
+
```

Comment:
Please delete all code that is commented out before merging.

Reply:
Thx, done.